code/modules/clothing/suits/f13factionarmor.dm

```diff
@@ -128,10 +128,11 @@
 	mob_overlay_icon = 'icons/fallout/onmob/clothes/armor_medium.dmi'
 
 /obj/item/clothing/suit/armor/f13/exile/ncrexile
-	name = "modified NCR armor"
-	desc = "A modified detoriated armor kit consisting of NCR gear and scrap metal."
-	icon_state = "ncrexile"
-	item_state = "ncrexile"
+	name = "NCR deserter vest"
+	desc = "A set of patrol armor with the NCR emblem removed and right pauldron taken off."
+	armor = list("melee" = 35, "bullet" = 23, "laser" = 13, "energy" = 20, "bomb" = 40, "bio" = 40, "rad" = 20, "fire" = 60, "acid" = 20)
```

Comment:
Remove this line of code, it's useless, it already has  **_armor = list("melee" = 35, "bullet" = 20, "laser" = 10, "energy" = 20, "bomb" = 40, "bio" = 40, "rad" = 20, "fire" = 60, "acid" =20)_** these armor stats inherited by it's parent right above it, see "base faction exile armor" yeah, all exile armor uses these stats.

Reply:
Saw this VERY late, will fix once added.
A bit odd though because the Legion exile armor has its own stats.

Reply:
> Saw this VERY late, will fix once added. A bit odd though because the Legion exile armor has its own stats.

I think that's the case because Legion armor in general get high melee and low bullet/laser as you can see in the stats below, you can do the same with the NCR if you wish (Giving them high bullet/laser, low melee) or something of the sort for dynamic balancing.